src/azul/queues.py

```diff
@@ -86,13 +90,30 @@
 @attrs.frozen(kw_only=True)
 class SQSMessage:
     body: JSON
+    # These fields are None when the message hasn't been sent yet, and not None
+    # after it's been received
```

Comment:
```suggestion
    # These two fields are only set in messages received
```

Reply:
I omitted the "two" because the comment only applies to one field in the commit that introduces it. The 2nd field is added in a subsequent commit.